rav1e_js/src/lib.rs

```diff
@@ -166,6 +169,66 @@ impl Canvas {
       .collect()
   }
 
+  fn data_nv12(&self) -> Vec<u8> {
+    let data = self.data_argb();
+
+    // ImageFormats
+    let src_format = ImageFormat {
+      pixel_format: PixelFormat::Argb,
+      color_space: ColorSpace::Lrgb,
+      num_planes: 1,
+    };
+
+    let dst_format = ImageFormat {
+      // PixelFormat::{I420, I444} panicks with 'NotEnoughData: Not enough data provided'
+      // Err from https://github.com/aws/dcv-color-primitives/blob/cbcf039/src/lib.rs#L855
+      pixel_format: PixelFormat::Nv12,
+      color_space: ColorSpace::Bt709,
```

Comment:
It would be good to tag the resulting video as bt709 using rav1e's color API.

Reply:
Thx. I will figure out what this means :+1:

Reply:
Is it critical to set this config option?
Because currently the functionality to set the `ColorDescription` from the js side is not ready. Thereby I would like to postpone it.